src/modules/peek/Peek.UI/PeekXAML/Views/TitleBar.xaml

```diff
@@ -6,20 +6,26 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-    xmlns:local="using:Peek.UI.Views"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    FlowDirection="{x:Bind TitleBarFlowDirection, Mode=OneWay}"
     mc:Ignorable="d">
 
     <Grid x:Name="TitleBarRootContainer" Height="48">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition x:Name="SystemLeftPaddingColumn" Width="0" />
+            <ColumnDefinition x:Name="DraggableColumn" Width="*" />
+            <ColumnDefinition x:Name="LaunchAppButtonColumn" Width="Auto" />
+            <ColumnDefinition x:Name="AppRightPaddingColumn" Width="8" />
+            <ColumnDefinition x:Name="PinButtonColumn" Width="Auto" />
+            <ColumnDefinition x:Name="SystemRightPaddingColumn" Width="180" />
+        </Grid.ColumnDefinitions>
         <Grid
             x:Name="AppIconAndFileTitleContainer"
+            Grid.Column="1"
             Margin="8,0"
```

Comment:
I think the left margin should be 12?

Reply:
We were both wrong, it should be 16 :). I'll add a PR to the docs to get it fixed

![image](https://github.com/microsoft/PowerToys/assets/9866362/bee5b6b1-0b47-43e2-a0e8-322ce6103c51)

Reply:
PR https://github.com/MicrosoftDocs/winapps-winrt-api/pull/103